<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogProfessor</class>
 <widget class="QDialog" name="dialogProfessor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>757</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Professor</string>
  </property>
  <widget class="QGroupBox" name="gbDialogProEditData">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1131</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit Data</string>
   </property>
   <widget class="QLineEdit" name="txtProfessorAddress">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbAdress">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtProfessorPhone">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>110</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbPhone">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone number:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbDegreeCodePro">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>180</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbDegree">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Degree code:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdDialogProActiveYes">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>240</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdDialogProActiveNo">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>240</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>240</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Active:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtProfessorEmail">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbGenderPro">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Female</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelPassword">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtProfessorName">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelProfessorName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Professor full name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProfessorId">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Professor Id:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtProfessorId">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>280</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <widget class="QRadioButton" name="rdbtnAddProfessor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Professor</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbtnUpdateProfessor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Professor</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbtnDeleteProfessor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Professor</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushBtnProfessor">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>320</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dynamic Button</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbDialogProContent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>1131</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Contents</string>
   </property>
   <widget class="QTableView" name="tvProfessor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1091</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
